assets/airports/rjaa.json

```diff
@@ -0,0 +1,897 @@
+                    ["hal",  1],
+                    ["jal", 34],
+                    ["qfa",  1],
+                    ["sjo",  4],
+                    ["ual", 10],
+                    ["fdx",  4],
+                    ["nca", 16],
+                    ["ups",  4],
+                    ["aic",  1],
+                    ["uae",  2],
+                    ["etd",  1],
+                    ["eva",  2],
+                    ["gia",  2],
+                    ["kal",  5],
+                    ["mas",  5],
+                    ["qtr",  1],
+                    ["sco",  3],
+                    ["sia",  2],
+                    ["apj",  4],
+                    ["vnl",  6]
+                ]
+        },
+        {
+            "type": 		"cyclic",
+            "route":        "BINKS.LAKES.RJAA",
+            "frequency":    3,
+            "altitude":     14000,
+            "speed":		230,
+            "offset":		2,
+            "airlines": [
+                    ["amx",  1],
+                    ["aza",  2],
+                    ["ana", 54],
+                    ["dal", 17],
+                    ["hal",  1],
+                    ["jal", 34],
+                    ["qfa",  1],
+                    ["sjo",  4],
+                    ["ual", 10],
+                    ["fdx",  4],
+                    ["nca", 16],
+                    ["ups",  4],
+                    ["aic",  1],
+                    ["uae",  2],
+                    ["etd",  1],
+                    ["eva",  2],
+                    ["gia",  2],
+                    ["kal",  5],
+                    ["mas",  5],
+                    ["qtr",  1],
+                    ["sco",  3],
+                    ["sia",  2],
+                    ["apj",  4],
+                    ["vnl",  6]
+                ]
+        },
+        {
+            "type": 		"cyclic",
+            "route":        "BINKS.TYLER.RJAA",
+            "frequency":    3,
+            "altitude":     14000,
+            "speed":		230,
+            "offset":		2,
+            "airlines": [
+                    ["amx",  1],
+                    ["aza",  2],
+                    ["ana", 54],
+                    ["dal", 17],
+                    ["hal",  1],
+                    ["jal", 34],
+                    ["qfa",  1],
+                    ["sjo",  4],
+                    ["ual", 10],
+                    ["fdx",  4],
+                    ["nca", 16],
+                    ["ups",  4],
+                    ["aic",  1],
+                    ["uae",  2],
+                    ["etd",  1],
+                    ["eva",  2],
+                    ["gia",  2],
+                    ["kal",  5],
+                    ["mas",  5],
+                    ["qtr",  1],
+                    ["sco",  3],
+                    ["sia",  2],
+                    ["apj",  4],
+                    ["vnl",  6]
+                ]
+        },
+        {
+            "type": 		"cyclic",
+            "route":        "ABBOT.ABBOT_SOUTH_A.RJAA",
```

Comment:
I'm unsure of this, as the STARs actually depend on runway usage. Since 
this issue isn't being fixed, I am unable to develop this much further.